public/app/plugins/panel/gauge/panelcfg.cue

```diff
@@ -29,6 +29,8 @@ composableKinds: PanelCfg: {
 					common.SingleStatBaseOptions
 					showThresholdLabels:  bool | *false
 					showThresholdMarkers: bool | *true
+					minVizWidth:          uint32 | *250
+					minVizHeight:         uint32 | *200
```

Comment:
maybe 100?  250 seems OK if we could set it only for *new* panels...

Reply:
I'm good with that - here is what that looks like in example (set to 100 vs 200 /250)
<img width="1500" alt="Screenshot 2023-08-02 at 6 42 13 PM" src="https://github.com/grafana/grafana/assets/22381771/aa6f24eb-a210-4d17-be29-ff1069f51a66">